transformation/src/jade-schema/tables/human_donor.table.json

```diff
@@ -18,8 +18,7 @@
     },
     {
       "name": "age",
-      "datatype": "string",
-      "type": "required"
+      "datatype": "string"
```

Comment:
Should age be a string? Or an integer?

Reply:
(It might need to become 2 columns: age and age_unit)

Reply:
Good catch! We do definitely need to store age unit information separately. I added the `age_unit` column and made `age` an integer.